lib/preneeds/middleware/response/eoas_xml_errors.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+module Preneeds
+  module Middleware
+    module Response
+      class EoasXmlErrors < Faraday::Response::Middleware
+        include SentryLogging
+        attr_reader :status, :fault, :code, :detail
+
+        def on_complete(env)
+          return unless env.response_headers['content-type'] =~ /\b(xml)/
```

Comment:
Won't it always be XML? I think we would want to return from this middleware based on status code instead maybe.